socket.go

```diff
@@ -24,14 +24,26 @@ var remote struct {
 	mu    sync.Mutex
 }
 
+//commandHook interface provides an override method signature usable by SocketPathHook and IsRunningHook
```

Comment:
This isn’t accurate anymore (IsRunningHook has a different signature), but can probably just removed, like the type in the other file.

Reply:
good point